src/assets/wise5/components/openResponse/speech-to-text/speech-to-text.component.ts

```diff
@@ -44,10 +44,10 @@ export class SpeechToTextComponent {
     { languageCode: 'pt-BR', language: $localize`Portuguese (Brazilian)` },
     { languageCode: 'es-US', language: $localize`Spanish` }
   ];
-  @ViewChild('languageSelectButton') languageSelectButton: MatButton;
   protected languages: Language[];
   @Output() newTextEvent: EventEmitter<string> = new EventEmitter<string>();
-  protected recording: boolean;
+  protected recording: Signal<boolean> = this.transcribeService.recording;
+  protected requestedRecording = false;
```

Comment:
I'm wondering if the requestedRecording variable name can be renamed to be more clear? It took me a while to figure out that this variable signifies that this component is the one that is recording as opposed to any recording currently being requested. Maybe if it's named recordingRequester or currentRecorder or something it would be better.

Reply:
Yah, good suggestion. I struggled with naming when I was working on this. I ended up going with recordingRequester.